packages/core/src/deploy.ts

```diff
@@ -69,6 +70,7 @@ export async function deploy<
   strategy?: StrategyT;
   strategyConfig?: StrategyConfig[StrategyT];
   maxFeePerGasLimit?: bigint;
+  maxPriorityFeePerGas?: bigint;
```

Comment:
Should this not be part of the `config: DeployConfig`. What is our rule for moving it up to the top level?

Reply:
`DeployConfig` is meant for configuring ignition's behavior. `maxFeePerGasLimit` and `maxPriorityFeePerGas` are not part of `DeployConfig` because they can be configured differently on a per network basis.